doc/sphinx/reference/vsm.rst

```diff
@@ -121,3 +121,45 @@ varnishd are no longer relevant across namespaces.
 
 To disable liveness checks based on PIDs, the variable ``VSM_NOPID`` needs to
 be present in the environment of VSM readers.
+
```

Comment:
There were two touchup commits to this one which I think we should also add (not critical)
f53bb248691a17849b46a4dbc897fa5b3861e000
5f650460977dd238af8e0d948a95caa85e617f2a

Reply:
Added both commits.